### PR TITLE
Pull Request for Portfolio Desktop version

### PR DESCRIPTION
**Pull request by anita00001**
In this branch, we have created a desktop version of the Portfolio website.

- Implemented Media query
- Implemented Responsive Website Design

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -483,3 +494,466 @@ footer .bottom_line {
...
+    background-image: none;
+  }
+
+  .works {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    background: #e5e5e5;
+    align-items: center;
+  }
```
Comment (GedeonTS) [inline style.css]: - [x] :thinking: In my opinion, you should stick to the design as much as possible by making sure the work section looks the same as the Figma design on the desktop version shown below :point_down: 

__You  will notice that some cards must only show the see project button on hover effect, you are not required to create the JavaScript interaction yet but, kindly make it display none with CSS on the desktop version for now__




your design:
Screenshot from 2023-01-20 06-14-02



Figma:
Screenshot from 2023-01-20 06-13-16

File: style.css [hunk 2]
```diff
@@ -483,3 +494,466 @@ footer .bottom_line {
...
+    line-height: 160%;
+    color: #f4f5f7;
+  }
+
+  .connect_with {
+    width: fit-content;
+    background-image: none;
+  }
+
+  .works {
```
Comment (GedeonTS) [inline style.css:604]: - [x] Kindly make sure your website is totally responsive on the desktop version, as you can see below the works section is crossing the container at some point, you can use percentages to make sure itis scaling correctly:
Screenshot from 2023-01-20 06-15-38

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @anita00001  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
- Good use of media queries :+1: 
- No linter errors :+1: 
- Descriptive README.md :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (wayungi) [reply style.css]: - [x]  Hello, You have done good work implementing the requested changes, however, there is a little more work still left to be done. I will break it down below.

- [x] According to the Figma design, the desktop version shows 3 projects in a row. Yours is showing only 2. Please make it show 3 projects on the desktop version in a row.
_**figma**_
proj-1
_**Your design**_
proj-2

- [x]  All Projects have a transparent dark overlay on top. when you hover over a project, the overlay should disappear and reveal the project with a **see project** button.
__**how a project should appear**_
pic-1

_**How a project should appear when hovered over**_
pic-2

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @anita00001 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
image



 ## Highlights
- [ ] Good PR title ✔️ 
- [ ] GitHub flow ✔️ 
- [ ] Descriptive commit message ✔️ 
- [ ] Professional README ✔️ 
- [ ] All linter checks passing

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — APPROVED** [review]
Hi @anita00001 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

CookedDelectableDolphin-size_restricted


## Highlights
All linters are passing :heavy_check_mark: 
Great effort in making the requested changes :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**